x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -81,6 +81,7 @@ public class XPackLicenseState {
                 "The CCR monitoring endpoint will be blocked",
                 "Existing follower indices will continue to replicate data" }
         );
+        messages.put(XPackField.REDACT_PROCESSOR, new String[] { "Executing a redact processor in an ingest pipeline will fail." });
```

Comment:
" unless the `skipIfUnlicensed` flag is set to true, in which case the redact processor will be skipped"?

Reply:
I'll do a bit of string review as a follow up -- the status quo in that file seems to favor pretty terse explanations, and I don't want to buck that, but I do also want to be truthful, etc.